.github/workflows/ci-core.yml

```diff
@@ -139,6 +199,7 @@ jobs:
           args: >
             -Dsonar.go.tests.reportPaths=${{ steps.sonarqube_report_paths.outputs.sonarqube_tests_report_paths }}
             -Dsonar.go.coverage.reportPaths=${{ steps.sonarqube_report_paths.outputs.sonarqube_coverage_report_paths }}
+            -Dsonar.go.golangci-lint.reportPaths=golangci-lint-report/golangci-lint-report.xml
```

Comment:
If there is/are no lint report/s, it would not block sonar workflow, those will be gracefully skipped, the only consequence is less information (that can raise an issue) in sonar reports